contributors/devel/e2e-tests.md

```diff
@@ -136,6 +136,14 @@ with this command:
 go run hack/e2e.go -- -v --down
 ```
 
+## Building the Tests
+
+* You can quickly recompile the e2e testing framework via `go install ./test/e2e`.
+  This will not do anything besides allow you to verify that the go code compiles.
+
+* If you want to run your e2e testing framework without re-provisioning the e2e setup,
```

Comment:
How is this different from `go run hack/e2e.go -- --test` (aka not having `--up` in your current or `--down` in your previous invocation)?